README.md

```diff
@@ -173,6 +179,98 @@ Whether or not to force push a reference with different commit history before th
 code-suggester pr -o foo -r bar -d 'description' -t 'title' -m 'message' --git-dir=.
 ```
 
+## Action
+
+### create a pull request
+Opens a GitHub Pull Request against the upstream primary branch with the provided git directory. By default the git directory is the same as the `$GITHUB_WORKSPACE` directory.
+
+#### Syntax
+
+`pr --upstream-repo=<string> --upstream-owner=<string> --title=<string> --description=<string> --message=<string> [options] `
+
+#### Environment Variables
+#### `ACCESS_TOKEN`
+*string* <br>
+**Required.** The GitHub access token which has permissions to fork, write to its forked repo and its branches, as well as create Pull Requests on the upstream repository. We recommend storing it as a secret in your GitHub repository.
+
+#### Options
+
+#### `--upstream-repo, -r`
+*string* <br>
+**Required.** The repository to create the fork off of.
+
+
+#### `--upstream-owner, -o`
+*string* <br>
+**Required.** The owner of the upstream repository.
+
+
+#### `--description, -d`
+*string* <br>
+**Required.** The GitHub Pull Request description.
+
+#### `--title, -t`
+*string* <br>
+**Required.** The GitHub Pull Request title.
+
+#### `--branch, -b`
+*string* <br>
+The GitHub working branch name. Default value is: `'code-suggestions'`.
+
+#### `--primary, -p`
+*string* <br>
+The primary upstream branch to open a PR against. Default value is: `'master'`.
+
+#### `--message, -m`
+*string* <br>
+**Required.** The GitHub commit message.
+
+#### `--force, -f`
+*boolean* <br>
+Whether or not to force push a reference with different commit history before the remote reference HEAD. Default value is: `false`.
+
+#### `--maintainers-can-modify, --modify`
+*boolean* <br>
+Whether or not maintainers can modify the pull request. Default value is: `true`.
+
+#### `--git-dir`
+*string* <br>
+**Required.** The path of a git directory. Relative to `$GITHUB_WORKSPACE`.
+
+### Example
+The following example is a `.github/workflows/main.yaml` file in repo `Octocat/HelloWorld`. This would add a LICENSE folder to the root `HelloWorld` repo on every pull request if it is not already there.
+```yaml
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: ci
+jobs:
+  add-license:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10, 12, 13]
+    env:
+      ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: googleapis/code-suggester@v1
+      - run: (curl http://www.apache.org/licenses/LICENSE-2.0.txt) > LICENSE
+      - uses: googleapis/code-suggester@v1
+        with:
+          args: >-
+            pr
+            -o Octocat
+            -r HelloWorld
+            -d 'This pull request is adding a LICENSE file'
+            -t 'chore(license): add license file'
+            -m 'chore(license): add license file'
+            -b 'my-branch'
+            --git-dir='.'
```

Comment:
I would prefer that we make these inputs:
```suggestion
        with:
          command: pr
          upstreamOwner: Octocat
          upstreamRepo: HelloWorld
          description: 'This pull request is adding a LICENSE file'
          title: 'chore(license): add license file'
          message: 'chore(license): add license file'
          branch: my-branch
          # maybe this defaults to '.'
          gitDir: '.'
```

Reply:
This would insulate any CLI argument changes from the action configuration

Reply:
If I do that then I think I will need a conditional `ENTRYPOINT` or `CMD` into my dockerfile and we will likely also need a bash script. Is that behaviour okay?